pages/download.js

```diff
@@ -106,14 +106,50 @@ const STYLES_BUTTON_PRIMARY = css`
   transition: 200ms ease all;
   user-select: none;
   cursor: pointer;
-  background-color: ${Constants.system.wall};
-  color: ${Constants.system.slate};
-  box-shadow: 0px 10px 50px 20px rgba(0, 0, 0, 0.1);
+  background-color: ${Constants.system.brand};
+  color: ${Constants.system.white};
+  box-shadow: -4px -4px 10px #ffffff, 4px 4px 10px #d4d4d4;
+
+  :hover {
+    background-color: ${Constants.system.brand};
+    box-shadow: -4px -4px 10px #ffffff, 4px 4px 10px #d4d4d4;
```

Comment:
this value doesn't change in the hover so theres no point in having it, same with the background-color.